src/api/authAPI.ts

```diff
@@ -0,0 +1,62 @@
+import type { FetchSignInInfoTypes } from '@/types/authTypes';
+
+const BASE_URL = process.env.NEXT_PUBLIC_KEYDEUK_API_BASE_URL;
+
+export const getCheckEmailDuplication = async (emailValue: string) => {
+  const url = `${BASE_URL}/api/v1/users/check/email?email=${emailValue}`;
+  try {
+    const response = await fetch(url);
+    const data = await response.json();
+    return data;
+  } catch (error) {
+    throw error;
+  }
+};
+
+export const getCheckNicknameDuplication = async (nickname: string) => {
+  const url = `${BASE_URL}/api/v1/users/check/nickname?nickname=${nickname}`;
+  try {
+    const response = await fetch(url);
+    const data = await response.json();
+    return data;
+  } catch (error) {
+    throw error;
+  }
+};
+
+export const postSignup = async (formData: FormData) => {
+  const url = `${BASE_URL}/api/v1/users`;
+
+  try {
+    const response = await fetch(url, {
+      method: 'POST',
+      headers: {
+        accept: 'application/json',
+      },
+      body: formData,
+    });
+    const data = await response.json();
+    return data;
+  } catch (error) {
+    throw error;
+  }
+};
+
+export const postSignin = async (formData: FetchSignInInfoTypes) => {
+  const url = `${BASE_URL}/login`;
+
+  try {
+    const response = await fetch(url, {
+      method: 'POST',
+      headers: {
+        'Content-Type': 'application/json',
+        Accept: 'application/json',
+      },
+      body: JSON.stringify(formData),
+    });
```

Comment:
```suggestion
      },
      body: JSON.stringify(formData),
    });
```
json으로 파싱해서 보내는거면 accept는 없어되지않아요?